corpus_transformation_scripts/Elexicon/Begriffe_eLexikon.xml

```diff
@@ -1,74 +1,108 @@
 	<lem elex="alligatio">alligatio</lem>
+	<lem elex="allodium">allodis</lem>
+	<lem elex="allodium">alodis</lem>
 	<lem elex="allodium">allodium</lem>
-	<lem elex="amanuensis">amanuensis</lem>
+	<lem elex="amanuensis">amanvensis</lem>
+	<lem elex="amanuensis">manvensis</lem>
 	<lem elex="ambasciare">ambasciare</lem>
+	<lem elex="ambasciare">ambascio</lem>
 	<lem elex="amicitia">amicitia</lem>
 	<lem elex="anathema">anathema</lem>
+	<lem elex="anathema">anathematizare</lem>
+	<lem elex="anathema">anathematizo</lem>
+	<lem elex="anathema">anathematus</lem>
+	<lem elex="anathema">anathemare</lem>
+	<lem elex="anathema">anathemo</lem>	
 	<lem elex="ancilla">ancilla</lem>
 	<lem elex="antrustio">antrustio</lem>
 	<lem elex="apostolicus">apostolicus</lem>
-	<lem elex="appennis">apennes</lem>
-	<lem elex="appennis">apennis</lem>
-	<lem elex="appennis">appennes</lem>
-	<lem elex="arrha">arrha</lem>
+	<lem elex="appennis">appennis</lem>
 	<lem elex="arrha">arra</lem>
 	<lem elex="auctor">auctor</lem>
-	<lem elex="auctor">auctor</lem>
 	<lem elex="auctoritas">auctoritas</lem>
+	<lem elex="auditores">auditor</lem>
 	<lem elex="auditores">auditores</lem>
 	<lem elex="basilica">basilica</lem>
+	<lem elex="basilica">Basilica_Ortsname</lem>
 	<lem elex="benedictio">benedictio</lem>
+	<lem elex="benedictio">benedictus</lem>
+	<lem elex="benedictio">benedicere</lem>
+	<lem elex="benedictio">benedico</lem>
 	<lem elex="beneficium">beneficium</lem>
 	<lem elex="boni_homines">boni homines</lem>
 	<lem elex="boni_homines">bonus homo</lem>
-	<lem elex="burgundi">burgundi</lem>
-	<lem elex="burgundi">burgundio</lem>
+	<lem elex="burgundi">Burgundia</lem>
+	<lem elex="burgundi">Burgundio</lem>
 	<lem elex="burgus">burgus</lem>
-	<lem elex="caespitaticum">caespitaticum</lem>
+	<lem elex="burgus">Burgus_Ortsname</lem>
+	<lem elex="caespitaticum">caespitaticus</lem>
 	<lem elex="camerarius">camerarius</lem>
+	<lem elex="camerarius">camerlanus</lem>
+	<lem elex="camerarius">camerlengus</lem>
+	<lem elex="camerarius">Camerlengus_Personenname</lem>
+	<lem elex="camerarius">subcamerarius</lem>
 	<lem elex="cancellarius">cancellarius</lem>
 	<lem elex="canonicus">canonicus</lem>
+	<lem elex="canonicus">cocanonicus</lem>
 	<lem elex="caritas">caritas</lem>
+	<lem elex="caritas">caritativus</lem>
 	<lem elex="carropera">carropera</lem>
-	<lem elex="carta">carta</lem>
+	<lem elex="carropera">carroperae</lem>
 	<lem elex="carta">charta</lem>
+	<lem elex="carta">chartula</lem>
+	<lem elex="carta">carta</lem>
+	<lem elex="carta">cartula</lem>
+	<lem elex="carta">cartola</lem>
 	<lem elex="cathedra">cathedra</lem>
 	<lem elex="causa">causa</lem>
+	<lem elex="causa">causella</lem>
 	<lem elex="cautio">cautio</lem>
 	<lem elex="cella">cella</lem>
+	<lem elex="cella">Cella_Ortsname</lem>
 	<lem elex="cellula">cellula</lem>
 	<lem elex="census">census</lem>
 	<lem elex="cessio">cessio</lem>
 	<!--<lem elex="centenarius">centenarius</lem>-->
-	<lem elex="chrisam">chrisam</lem>
```

Comment:
"chrisam" also Wort taucht aktuell in der Werkstatt nicht auf. Es könnte aber geben, dass das Wort in eine Übersetzung oder E-Lexikonartikel irgendwann erscheint. Von daher würde ich die Zeile mit "chrisam"->"chrisam" stehen lassen.